inputs/dust/SI_strat1_amr_BA.in

```diff
@@ -0,0 +1,146 @@
+# ========================================================================================
+#  (C) (or copyright) 2023-2024. Triad National Security, LLC. All rights reserved.
+#
+#  This program was produced under U.S. Government contract 89233218CNA000001 for Los
+#  Alamos National Laboratory (LANL), which is operated by Triad National Security, LLC
+#  for the U.S. Department of Energy/National Nuclear Security Administration. All rights
+#  in the program are reserved by Triad National Security, LLC, and the U.S. Department
+#  of Energy/National Nuclear Security Administration. The Government is granted for
+#  itself and others acting on its behalf a nonexclusive, paid-up, irrevocable worldwide
+#  license in this material to reproduce, prepare derivative works, distribute copies to
+#  the public, perform publicly and display publicly, and to permit others to do so.
+# ========================================================================================
+
+<artemis>
+problem = SI_strat  # name of the pgen
+coordinates = cartesian
+
+<parthenon/job>
+problem_id = runBA  # problem ID: basename of output filenames
+
+<parthenon/output0>
+file_type = hst
+dt 	  = 0.01
+
+<parthenon/output1>
+variables = gas.prim.density,   &
+            gas.prim.velocity,  &
+            dust.prim.density, &
+            dust.prim.velocity
+
+file_type = hdf5  # HDF5 data dump
+dt        = 2.0  # time increment between outputs
+
+<parthenon/output2>
+file_type = rst
+dt        = 100.0
+
+<parthenon/time>
+nlim       = -1     # cycle limit
+tlim       = 400.0  # time limit
+integrator = rk2    # time integration algorithm
+ncycle_out = 100    # interval for stdout summary info
+
+<parthenon/mesh>
+nx1    = 1024    # Number of zones in X1-direction
+x1min  = -0.4   # minimum value of X1
+x1max  =  0.4   # maximum value of X1
+ix1_bc = extrapolate   # Inner-X1 boundary condition flag
+ox1_bc = extrapolate   # Outer-X1 boundary condition flag
+
+nx2    = 512    # Number of zones in X2-direction
+x2min  = -0.2   # minimum value of X2
+x2max  =  0.2   # maximum value of X2
+ix2_bc = extrapolate   # Inner-X2 boundary condition flag
+ox2_bc = extrapolate   # Outer-X2 boundary condition flag
+
+nx3    = 1      # Number of zones in X3-direction
+x3min  = -0.5   # minimum value of X3
+x3max  =  0.5   # maximum value of X3
+ix3_bc = periodic   # Inner-X3 boundary condition flag
+ox3_bc = periodic   # Outer-X3 boundary condition flag
+
+# nghost = 2
+refinement = static
+numlevel = 2		#3
+
+<parthenon/static_refinement1>
+x1min = -0.3
+x1max =  0.3
+x2min = -0.02
+x2max =  0.02
+level = 1
+
+
```

Comment:
remove extraneous whitespace here and elsewhere.

Reply:
this input is removed in the new code